src/js/apps/patients/patient/data-events/data-events_app.js

```diff
@@ -16,10 +16,11 @@ export default App.extend({
     return [
       Radio.request('entities', 'fetch:actions:collection:byPatient', { patientId: patient.id, filter }),
       Radio.request('entities', 'fetch:flows:collection:byPatient', { patientId: patient.id, filter }),
+      Radio.request('entities', 'fetch:patientEvents:collection', patient.id),
```

Comment:
If need be later we might have to come back and change this to `_DEVELOP_ ? { models: [] } " Radio.request....`   but let's see how things go.